index.html

```diff
@@ -217,8 +217,6 @@ <h1 id="ovningar">Övningar</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="vendor/highlight.pack.js"></script>
     <script src="vendor/bootstrap.min.js"></script>
-    <script src="vendor/smooth-scroll.js"></script>
```

Comment:
OK, yeah, attaching event listeners. Not only on anchor links, but that's cool.

Yeah, it exists, but I mean, marquee and blink was popular in the 90s.

So, currently, you're blocking rendering with 2 stylesheets which are fetched from a remote URI. jQuery is also fetched remotely. We've got exercises.zip, exercises.json, package, import, import.py, exercise-files/, exercises/, and of course the other repo exercises with another exercises/. We've got 3 different versions of site.js at this point, getting data from different locations. Then we've got the servers, where we've got the gulp one, as well as the ruby one. The workflow here is not really... obvious. Each developer here has his own view on how this site is compiled and where data is gathered from.

Will it matter? Hard to say. You can do a lot of wierd stuff without it mattering to the audience.